### PR TITLE
Add test deployment scripts

### PR DESCRIPTION
**Pull request by Arshia001**

### CODE REVIEW

File: marketplace/scripts/cmd-deploy-empty.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    util.run("anchor build");
+
+    let sessionName = `mu_marketplace_${Date.now()}`;
+    console.log(`Starting tmux session ${sessionName}`);
+    console.log("Starting local Solana validator");
+    let tmuxSession = new TmuxSession(sessionName, "solana-test-validator");
+
+    console.log("Waiting for validator to start");
+    util.waitUntilPortUsed(8899);
+    // Wait an additional 2 seconds for the node to become healthy
```
Comment (hamidrezakp) [inline marketplace/scripts/cmd-deploy-empty.ts:16]: you can call `get_last_blockhash` which returns when there is at least one block.
and the node is healthy.
Current method might not work on slower systems like ci/cd

**Review by hamidrezakp — CHANGES_REQUESTED** [review]
why do you have this 2mb binary?
marketplace/scripts/test-stack/test-function.wasm

build it before tests.

Arshia: Done

**Comment by Arshia001** [comment]
Remove tmux :D

**Comment by hamidrezakp** [comment]
> Remove tmux :D

This is my lucky day :D thanks

**Pull request merged**